docs/tutorials/csharp/basic_csharp.md

```diff
@@ -1,7 +1,6 @@
 ---
 nav_exclude: true 
 ---
-
```

Comment:
Should this one be in the nav too?

Reply:
It didn't look complete and there is already the get started with csharp, so this seemed like a duplicate, but I didn't want to delete it. Should it be added to the get started section? Or if you think it makes sense to add it in this section, I am good with that too!

Reply:
This was has had so few views, I think we can safely delete it

Reply:
As discussed will and pull in the content later